sympa.spec

```diff
@@ -251,7 +251,11 @@ Provides:    bundled(js-jquery) = 3.2.1
 %endif
 # jquery-migrate
 %if %{unbundle_jquery_migrate}
+%if %{?el7}
+Requires:    python-XStatic-JQuery-Migrate
+%else 
 Requires:    xstatic-jquery-migrate-common
+%endif
 %else
 Provides:    bundled(js-jquery-migrate) = 1.4.1
 %endif
```

Comment:
jquery-migrate is currently not unbundled. Fedora and EL7 ships an older version than sympa (1.2.1 vs 1.4.1). There's no package at all for EL6. Anyway, this fix is fine to commit.